eks-service-definition.yml

```diff
@@ -81,6 +85,10 @@ provision:
     type: number
     default: 1
     overwrite: true
+  - name: use_hardened_ami
+    type: boolean
+    default: false
```

Comment:
Why is default set to `false` here?

Reply:
Nice question 😃 It's false because the code originally came from a team that did not yet have access to the Hardened AMI and since this repo is to serve the public first and data.gov second, the default is to make it work in the most reliable way and make security an increased feature for those with the required access.

Reply:
Ok, that makes more sense. I was trying to parse the disparity between this and the default: true in the vars file.

Reply:
I didn't even realize that the terraform variable was default to true and the broker variable was default to false.  That's a good catch.  I guess the terraform value should also be set to false.

Reply:
I think?  Right?

Reply:
Yeah, I see what you're saying.  The `variables.tf` file is just straight terraform that anyone can run and bring up a cluster.  The `eks-service-definition.yml` packages the terraform code as a portable utility that can be run in a cloudfoundry-type environment.  I think the "right thing" is to set the terraform code to false by default as well (and update the README) because the terraform code can be also references a terraform module which can be called by anyone and therefore should be accommodating of the fact that anyone would not necessarily have the Hardened AMI.

Since no one is using the repo as a terraform module, the "right thing" is irrelevant.  If we have to use the default AMI because of Solr issues, setting the default to `False` in the terraform `variables.tf` would help us, but instead of making the change now, I'll defer to when we know we can't use the Hardened AMI for Solr.  (I don't think we'll be able to use the Hardened AMI because of Solr.)

Please let me know if I have to elaborate on any of that because it was a mess and I'm sorry about the mess 😢

Reply:
I like to call my messes "works in progress", it makes them sound more official. 

Super clear explanation. What do we then modify if we want to pass in `true` in our case?

Reply:
As a brokerpak? Or with our statically managed EKS deployment?

Reply:
Both, really. We can cover this in pairing.

Reply:
Awesome. That helps a ton. Thanks!